examples/CMakeLists.txt

```diff
@@ -2,37 +2,43 @@
 # Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-if(UMF_BUILD_LIBUMF_POOL_SCALABLE)
-    set(EXAMPLE_NAME umf_example_basic)
+# NOTE: the basic example is always built, even if the TBB required by the
+# Scalable Pool is not detected by CMake. If the TBB is not detected during the
+# execution of the basic example, the user will receive a corresponding error
```

Comment:
This message won't be displayed (nor TBB will be detected) during the execution of the example, rather during the configuration phase of the project.

Reply:
Yes - there is one message during configuration and second message during execution: "... required by Scalable Pool not found - install TBB malloc or make sure it is in the default search paths"